pravega/pravega
# Issue 326: Moving metrics to shared

**Change log description**
We have been discussing issue #326 for a long time, and I'm currently inclined to do this in stages. As part of determining the feasibility of doing it in stages, I've moved metrics out of `common` and into a project called `shared`.  There is no functional difference compared to the existing code base.

I have also renamed the package to eliminate `common`. I don't think we should add `shared` to the topic name, and I've left it as `com.emc.pravega.metrics`.

**Purpose of the change**
Move metrics classes and interfaces to `shared`.

**What the code does**
Move internal code around, there is no API change.

**How to verify it**
There is no functional change. One way to verify is to pull the diff and build it.
